src/sdk/s3.ts

```diff
@@ -64,6 +80,7 @@ class S3Service {
     if (!bucket) throw new Error("Bucket not specified");
     const response = await this.s3.listObjectsV2({
       Bucket: bucket,
+      Prefix: `${dir ? dir : ""}`,
```

Comment:
@gurpinder11 It does not work on my system without adding this. Please also confirm that this is working for you.

If this does not work for you, we'll have to dig a little deeper and see why is there an inconsistency in the behaviour between systems.